src/layouts/AboutLayout.js

```diff
@@ -0,0 +1,338 @@
+        helpful:
+      </div>
+      <RequiredSkills>
+        <li>Front end development (react)</li>
+        <li>Back end development (node, postgres)</li>
+        <li>Webdesign</li>
+        <li>Marketing (social media marketing, creation of an image video)</li>
+        <li>Knowledge in the assessment of CO2-footprints of processes / knowledge in LCA</li>
+        <li>Generally experience with founding and working in a non-profit organisation</li>
+      </RequiredSkills>
+    </Container>
+  );
+}
+
+const InfoBubble = ({ data }) => (
+  <InfoBubbleContainer>
+    <Bubble>
+      <IconWrapper>
+        <i className={"fas " + data.icon}></i>
+      </IconWrapper>
+    </Bubble>
+    <h3>{data.title}</h3>
+    <InfoText>{data.text}</InfoText>
+  </InfoBubbleContainer>
+);
+
+const Member = ({ data }) => (
+  <MemberInfo>
+    <img src={data.image}></img>
+    <h2>{data.name}</h2>
+    <Location>
+      <i className="fas fa-map-marker-alt"></i>
+      {data.location}
+    </Location>
+  </MemberInfo>
+);
+
+const Container = styled.div`
+  color: ${({ theme }) => theme.colors.primary};
+  text-align: center;
+  h1 {
+    font-weight: 500;
+    font-size: 52px;
+    margin-bottom: 50px;
+    line-height: 60px;
+  }
+  h2 {
+    font-size: 35px;
+  }
+  @media (max-width: 900px) {
+    h1 {
+      font-size: 30px;
+    }
+    h2 {
+      font-size: 25px;
+    }
+  }
+`;
+
+const AboutImageContainer = styled.div`
+  width: 100%;
+  height: 373px;
+  background-image: url("/about_background.png");
+  background-position: center;
+  background-size: cover;
+  margin-bottom: 75px;
+`;
+
+const HeadlineContainer = styled.div`
+  display: table;
+  height: 100%;
+  width: 100%;
+`;
+
+const Headline = styled.div`
+  display: table-cell;
+  vertical-align: middle;
+  width: 100%;
+`;
+
+const HeadlineText = styled.div`
+  color: white;
+  font-size: 85px;
+  border: 5px solid white;
+  display: inline-block;
+  margin: 0 auto;
+  padding-top: 3%;
+  padding-bottom: 3%;
+  padding-right: 10%;
+  padding-left: 10%;
+  @media (max-width: 1075px) {
+    font-size: 75px;
+    padding-right: 5%;
+    padding-left: 5%;
+  }
+  @media (max-width: 875px) {
+    font-size: 50px;
+    padding-right: 5%;
+    padding-left: 5%;
+  }
+  @media (max-width: 575px) {
+    font-size: 40px;
+    padding-right: 5%;
+    padding-left: 5%;
+  }
+`;
+
+const BubbleGrid = styled.div`
+  margin-bottom: 3rem;
+  max-width: 1390px;
+  margin: 0 auto;
+  margin-top: 50px;
+  @media (max-width: 1000px) {
+    grid-template-columns: 1fr 1fr;
+  }
+  @media (max-width: 600px) {
+    grid-template-columns: 1fr;
+  }
+`;
+
+const InfoBubbleContainer = styled.div`
+  display: inline-block;
+  vertical-align: top;
+  padding-left: 50px;
+  padding-right: 50px;
+  h3 {
+    font-size: 35px;
+  }
+  margin-bottom: 50px;
+`;
+
+const Bubble = styled.div`
+  border: 1px solid black;
+  border-radius: 100%;
+  width: 150px;
+  height: 150px;
+  margin: auto;
+  display: table;
+`;
```

Comment:
Styling on the bubble and grid seems a bit weird, could probably be 3 or 4 in a row at fullscreen and the text should probably be split up some or made more concise and have a larger line-height. Just a big block of text for a the format it's in.